apps/3000-home/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@module-federation/3000-home",
+  "version": "0.0.1",
+  "dependencies": {
+    "@nrwl/workspace": "14.7.3",
+    "fast-glob": "^3.2.12",
+    "next": "12.2.5",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "typescript": "~4.7.2"
+  },
+  "devDependencies": {
```

Comment:
It's strange that `@module-federation/nextjs-mf` package is not listed in devDependencies

Reply:
Fixed 👍